### PR TITLE
Fix filter of user consents

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #28497 .

### Summary of Changes
Fix filter for ordering consents.

### Testing Instructions
As described in #28497

### CODE REVIEW

**Comment by Kostelano** [comment]
I applied the patch, but the problem remains :)

Screenshot_3

**Comment by chmst** [comment]
@Kostelano could you re-test please? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28502">issues.joomla.org/tracker/joomla-cms/28502</a>.</sub>

**Comment by Kostelano** [comment]
Now everything works, thanks!

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on c23067990a748cdeebe03ed4c49f4015fe9cb427<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28502">issues.joomla.org/tracker/joomla-cms/28502</a>.</sub>

**Comment by chmst** [comment]
@Kostelano could set the test result with the "test this" Button in the issue tracker? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28502">issues.joomla.org/tracker/joomla-cms/28502</a>.</sub>

File: administrator/components/com_privacy/models/consents.php [hunk 1]
```diff
@@ -30,7 +30,9 @@ public function __construct($config = array())
 		if (empty($config['filter_fields']))
 		{
 			$config['filter_fields'] = array(
-				'id', 'a.id', 'a.user_id',
+				'id', 'a.id', 
+				'user_id', 'a.user_id',
+				'subject','a.subject',
```
Comment (Quy) [inline administrator/components/com_privacy/models/consents.php]:
```suggestion
				'subject', 'a.subject',
```

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on eff0ad2c6c709243a8fa04036a19e537d54c3233<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28502">issues.joomla.org/tracker/joomla-cms/28502</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28502">issues.joomla.org/tracker/joomla-cms/28502</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**